assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -36,6 +36,7 @@
 
 	&__preview {
 		height: 100%;
+		padding-bottom: 42.8571%; // BC: Fix for older browsers that don't support "aspect-ratio: 21 / 9;"
```

Comment:
@rami-elementor please add TODO: Remove this line in after we will deprecate Safari v13

Reply:
Done.